reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -22,34 +22,34 @@
   } ],
   "licenses" : [ {
     "_id" : 0,
-    "id" : "GPL-2.0-only WITH Classpath-exception-2.0 AND BSD-3-Clause OR MIT"
+    "id" : "GPL-2.0-only WITH Classpath-exception-2.0"
```

Comment:
Similar to the question in the previous commit, can the web app handle such licenses?

Reply:
It depends what you mean by handled. The WebApp treats 'GPL-2.0-only WITH Classpath-exception-2.0' as any other single license ID string.